src/Auth/AbstractPasswordHasher.php

```diff
@@ -1,4 +1,5 @@
 <?php
+declare(strict_types = 1);
```

Comment:
The [examples](https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#example) in proposed PSR12 don't show spaces around `=`. So it's better to follow that instead of having to potentially change in future.

Reply:
Also, couldnt a new sniffer rule also assert that?
Additionally, why not in the same line, no need to further space things
```
<?php declare(strict_types = 1);

/**
```

Reply:
The example further down also shows that this is valid.

Reply:
The example also states out: 
```
When wishing to declare strict types in files containing markup outside PHP opening and closing tags MUST, on the first line, include an opening PHP tag, the strict types declaration and closing tag.
```

Reply:
The case where strict types declaration is on same line as PHP tag is when the file has PHP code mixed with HTML. In PHP only files it's on line below the opening PHP tag, so let's keep the same.